mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -262,9 +267,16 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 		ARG 1 min
 		ARG 2 max
 	METHOD method_32751 nextBetween (Ljava/util/Random;II)I
+		COMMENT {@return a random, uniformly distributed integer value between {@code
+		COMMENT min} and {@code max}}
```

Comment:
Can we use math notations like `value in {@code [min, max]}` where `[]` indicates inclusive ranges while `()` indicates exclusive ranges?